src/components/body/BodyRow.ts

```diff
@@ -38,22 +38,19 @@ import { DataTableBodyCell } from './BodyCell';
       </div>
     </div>
   `,
-  directives: [ DataTableBodyCell ],
-  host: {
-    '[class.active]': 'isSelected'
-  }
+  directives: [ DataTableBodyCell ]
 })
 export class DataTableBodyRow {
 
   @Input() row: any;
 
-  get isSelected() {
-    return this.state.selected &&
-      this.state.selected.indexOf(this.row) > -1;
+  constructor(private state: StateService) {
   }
 
-  constructor(private state: StateService, elm: ElementRef) {
-    elm.nativeElement.classList.add('datatable-body-row');
+  @HostBinding('class.datatable-body-row') isBodyRow: boolean = true;
```

Comment:
I guess this can be pretty easy optimized by the angular compiler in the upcoming rc5. But u're right, would be nice to hear feedback from the core team.